code/game/machinery/computer/cloning.dm

```diff
@@ -358,35 +358,31 @@
 		return
 	if(scan_brain && !can_brainscan())
 		return
-	if((isnull(subject)) || (!(ishuman(subject))) || (!subject.dna) || (NO_SCAN in subject.dna.species.species_traits))
+	if(isnull(subject) || (!(ishuman(subject))) || (!subject.dna))
```

Comment:
This should really be split into 3 separate checks, with 3 separate errors:
Error: no subject in scanner
Error: subject is not humanoid
Error: subject has no DNA

Reply:
To be honest, I wasn't even sure how most of these conditions would ever happen.  The scan button is disabled if there's no subject inside the scanner.  You can't put in animals (non-humans?) into the scanner, and I'm not sure how someone could have no DNA.  

Any suggestions on how these things could happen so I can test them individually and add messages?

Reply:
There might be certain species without DNA, like for example IPC, possibly, or slimes.

I also wonder if you can put a simple_animal into the scanner.

Fair enough if you can't, but if the reason is "you put an IPC into the scanner, they have no DNA, they're not even organic" then that should be obvious from the error message.

Reply:
Simple animals I tried and it didn't let me put them in.  You make a good point about IPCs and slimes and I'll check to see if I need to add another message for them or if they already fall under the "[race] is not scannable" message.

Reply:
Think xenomorphs can be shoved in

Reply:
Will check that too

Reply:
Checked Xenomorphs and they are hitting the !ishuman condition.  Going to adjust messages to give more specific messages for IPCs and Xenos. Will do today